external/storm-druid/src/main/java/org/apache/storm/druid/bolt/TupleDruidEventMapper.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.storm.druid.bolt;
+
+import org.apache.storm.tuple.ITuple;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Converts {@link ITuple} to Event
+ */
+public final class TupleDruidEventMapper<E> implements ITupleDruidEventMapper {
+    private static final Logger LOG = LoggerFactory.getLogger(TupleDruidEventMapper.class);
+
+    public static final String DEFAULT_FIELD_NAME = "event";
+
+    public TupleDruidEventMapper() {
+    }
+
+    @Override
+    public E getEvent(ITuple tuple) {
+        return (E) tuple.getValueByField(DEFAULT_FIELD_NAME);
```

Comment:
You may want to take the field name in constructor and use it like `tuple.getValueByField(eventFieldName);`. Otherwise, user should always need to extend this class and override `getEvent(ITuple)` method.